examples/cluster/plot_digits_linkage.py

```diff
@@ -35,15 +36,18 @@
 
 np.random.seed(0)
 
+
+def shift(x):
+    return ndimage.shift(x.reshape((8, 8)),
+                  .3 * np.random.normal(size=2),
+                  mode='constant').ravel()
```

Comment:
it fits on two but not on one, I just thought it looked a bit cleaner on 3; I see why using two would be better though. The indentation seems to be rendering oddly here, it's pep8 compliant however.